english/blackAndwhite/blackAndwhite.md

```diff
@@ -13,7 +13,14 @@ Christy walks back to her bed and slide into her quilt.
 Along the direction of the wind, through the window next to the bed, A tall and toned young man, 
 wearing a leather hood and jeans, smirks seeing Christy tremble in fear.
 
-Christy sprints to the window to check if someone is hiding there and sees no one. Stranger drifts into the eternal dark
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.